openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -15,44 +15,46 @@ <h4 class="question__title ui-toggle-visibility__control">
             </h4>
 
             <div class="ui-toggle-visibility__content">
-                <ol class="question__answers">
-                    {% for option in criterion.options %}
-                    <li class="answer">
-                        <div class="wrapper--input">
-                            <input type="radio"
-                                   name="{{ criterion.name }}"
-                                   id="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}"
-                                   class="answer__value"
-                                   value="{{ option.name }}"
-                                   aria-labelledby="{{ rubric_type }}__assessment__rubric__prompt--{{ criterion.order_num }}"/>
-                            <label for="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}"
-                                   class="answer__label"
-                                   >{{ option.label }}</label>
-                        </div>
-                        <div class="wrapper--metadata">
-                            <span class="answer__tip">{{ option.explanation }}</span>
-                            <span class="answer__points">{{ option.points }} <span class="answer__points__label">{% trans "points" %}</span></span>
+                <div class="question__answers">
```

Comment:
@cptvitamin recommended removing ol/li structure here and using divs instead.

Also moved aria-labelledby to the group div (from individual inputs) and changed "name" attribute to "data-criterion-name", as name attribute should not have spaces in it (and criterion.name is specified by staff in Studio settings).